test/helpers/executeLIP36Upgrade.js

```diff
@@ -0,0 +1,35 @@
+const BondingManager = artifacts.require("BondingManager")
+const LinkedList = artifacts.require("SortedDoublyLL")
+const AdjustableRoundsManager = artifacts.require("AdjustableRoundsManager")
+
+const {contractId} = require("../../utils/helpers")
+import BN from "bn.js"
+
+module.exports = async function(controller, roundsManager, bondingAddress) {
```

Comment:
Worth considering a more descriptive var name than "bondingAddress" - maybe "newBondingManagerAddr"?

Reply:
let's roll with `bondingManagerProxyAddress` then that's most accurate.

Reply:
fixed in e3284a4